#1163: Tests for org.eclipse.kura.wire.component.provider

djurod opened the pull request:
- Prepared unit tests for WireAsset, DbWireRecordFilter, EmitJob and
  Timer classes.
- Changed Timer class so that it is possible to unit test it.

Signed-off-by: Djuro Drljaca <djuro.drljaca@comtrade.com>

amitjoy commented on kura/test/org.eclipse.kura.wire.component.provider.test/build.properties:
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2017 Eurotech and/or its affiliates
```
"... and others"

amitjoy commented on kura/test/org.eclipse.kura.wire.component.provider.test/pom.xml:
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2017 Eurotech and others
```
Copyright (c) 2017 Eurotech and/or its affiliates and others

amitjoy commented on kura/test/org.eclipse.kura.wire.component.provider.test/META-INF/MANIFEST.MF:
```diff
@@ -0,0 +1,17 @@
+Import-Package: org.junit;version="4.12.0",
+ org.junit.runners;version="4.12.0",
+ org.slf4j;version="1.6.4",
+ org.eclipse.kura.core.testutil,
```
Will there be any specific version of the package?

amitjoy commented on kura/test/org.eclipse.kura.wire.component.provider.test/META-INF/MANIFEST.MF:
```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.wire.component.provider.test
```
Wouldn't it be good if we provide more user friendly name to `Bundle-Name`? (such as `Wire Component Provider Tests`)

Commit:
#1163: Tests for org.eclipse.kura.wire.component.provider

- Prepared unit tests for WireAsset, DbWireRecordFilter, EmitJob and
  Timer classes.
- Changed Timer class so that it is possible to unit test it.

Signed-off-by: Djuro Drljaca <djuro.drljaca@comtrade.com>

djurod commented:
I think it should be OK now,
